[2024-12-26T16:34:19.497+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-26T16:34:19.523+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: docker_hello_world.docker_command scheduled__2024-07-31T00:00:00+00:00 [queued]>
[2024-12-26T16:34:19.534+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: docker_hello_world.docker_command scheduled__2024-07-31T00:00:00+00:00 [queued]>
[2024-12-26T16:34:19.535+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2024-12-26T16:34:19.553+0000] {taskinstance.py:2889} INFO - Executing <Task(DockerOperator): docker_command> on 2024-07-31 00:00:00+00:00
[2024-12-26T16:34:19.565+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=958) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-26T16:34:19.566+0000] {standard_task_runner.py:72} INFO - Started process 962 to run task
[2024-12-26T16:34:19.568+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'docker_hello_world', 'docker_command', 'scheduled__2024-07-31T00:00:00+00:00', '--job-id', '400', '--raw', '--subdir', 'DAGS_FOLDER/dag_dockeroperator.py', '--cfg-path', '/tmp/tmpcq1oncia']
[2024-12-26T16:34:19.573+0000] {standard_task_runner.py:105} INFO - Job 400: Subtask docker_command
[2024-12-26T16:34:19.658+0000] {task_command.py:467} INFO - Running <TaskInstance: docker_hello_world.docker_command scheduled__2024-07-31T00:00:00+00:00 [running]> on host 1484ae97611b
[2024-12-26T16:34:19.795+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='docker_hello_world' AIRFLOW_CTX_TASK_ID='docker_command' AIRFLOW_CTX_EXECUTION_DATE='2024-07-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-31T00:00:00+00:00'
[2024-12-26T16:34:19.796+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-26T16:34:19.848+0000] {docker.py:379} INFO - Starting docker container from image hello-world
[2024-12-26T16:34:19.855+0000] {docker.py:387} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2024-12-26T16:34:20.179+0000] {docker.py:73} INFO - 
[2024-12-26T16:34:20.179+0000] {docker.py:73} INFO - Hello from Docker!
[2024-12-26T16:34:20.180+0000] {docker.py:73} INFO - This message shows that your installation appears to be working correctly.
[2024-12-26T16:34:20.180+0000] {docker.py:73} INFO - 
[2024-12-26T16:34:20.180+0000] {docker.py:73} INFO - To generate this message, Docker took the following steps:
[2024-12-26T16:34:20.180+0000] {docker.py:73} INFO -  1. The Docker client contacted the Docker daemon.
[2024-12-26T16:34:20.181+0000] {docker.py:73} INFO -  2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
[2024-12-26T16:34:20.181+0000] {docker.py:73} INFO -     (amd64)
[2024-12-26T16:34:20.181+0000] {docker.py:73} INFO -  3. The Docker daemon created a new container from that image which runs the
[2024-12-26T16:34:20.181+0000] {docker.py:73} INFO -     executable that produces the output you are currently reading.
[2024-12-26T16:34:20.182+0000] {docker.py:73} INFO -  4. The Docker daemon streamed that output to the Docker client, which sent it
[2024-12-26T16:34:20.182+0000] {docker.py:73} INFO -     to your terminal.
[2024-12-26T16:34:20.182+0000] {docker.py:73} INFO - 
[2024-12-26T16:34:20.182+0000] {docker.py:73} INFO - To try something more ambitious, you can run an Ubuntu container with:
[2024-12-26T16:34:20.183+0000] {docker.py:73} INFO -  $ docker run -it ubuntu bash
[2024-12-26T16:34:20.183+0000] {docker.py:73} INFO - 
[2024-12-26T16:34:20.183+0000] {docker.py:73} INFO - Share images, automate workflows, and more with a free Docker ID:
[2024-12-26T16:34:20.183+0000] {docker.py:73} INFO -  https://hub.docker.com/
[2024-12-26T16:34:20.183+0000] {docker.py:73} INFO - 
[2024-12-26T16:34:20.184+0000] {docker.py:73} INFO - For more examples and ideas, visit:
[2024-12-26T16:34:20.184+0000] {docker.py:73} INFO -  https://docs.docker.com/get-started/
[2024-12-26T16:34:20.488+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-26T16:34:20.489+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=docker_hello_world, task_id=docker_command, run_id=scheduled__2024-07-31T00:00:00+00:00, execution_date=20240731T000000, start_date=20241226T163419, end_date=20241226T163420
[2024-12-26T16:34:20.543+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-26T16:34:20.584+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-26T16:34:20.586+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
